api/v1beta1/gitrepository_types.go

```diff
@@ -49,11 +49,13 @@ type GitRepositorySpec struct {
 	Interval metav1.Duration `json:"interval"`
 
 	// The timeout for remote git operations like cloning, default to 20s.
+	// +kubebuilder:default:=20
```

Comment:
I find it more likely that the following would work:

```suggestion
	// +kubebuilder:default:=20s
```

Reply:
Seems to make things work, I will write some tests to verify this tomorrow. I guess I only have myself to blame thinking this was a quick in and out fix.